modules/ROOT/pages/push/coding.adoc

```diff
@@ -57,17 +174,52 @@ func application(_ application: UIApplication,
 
 ****
 
+*Using plain Cordova*
+
 [source,javascript]
 ----
+const push = PushNotification.init({
+    android: {},
+    ios: {
+        alert: "true",
+        badge: "true",
+        sound: "true"
+    }
+});
+
 // Registration handler
-pushObject.on('registration').subscribe(data => {
-  // PushRegistration will handle the device registration with the Unifiedpush Server
-  new PushRegistration().register(data.registrationId).then(() => {
-    // Registration with UPS successful, you can now send push notifications from the UPS UI
-  }).catch(err => {
-    // Error on device registration
-  });
+push.on('registration', data => {
+    // PushRegistration is part of @aerogear/push
+    new PushRegistration().register(data.registrationId).then(() => {
+        // Registration with UPS successful, you can now send push notifications from the UPS UI
+    }).catch(err => {
+        // Error on device registration
+    });
 });
 ----
 
+*Using Ionic*
```

Comment:
I would be in favor of removing this. We provide a plugin for JS/Cordova, it's only the showcase where this framework is used but it could perfectly change.

Reply:
The thing is, there are some special requirements when you use Ionic with regards to the push plugin. Why not document them when we already know them?